black-mulberry-data-transfer/src/main/java/com/black/mulberry/data/transfer/request/ProductCommentRequest.java

```diff
@@ -4,13 +4,22 @@
 import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Range;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
 @Data
 public class ProductCommentRequest {
 
+    @NotNull(message = "product id is not null")
```

Comment:
![image](https://user-images.githubusercontent.com/84815716/203625632-4c27867e-0df9-4775-9e9a-0b9cbf53ded6.png)

https://stackoverflow.com/questions/26638047/how-to-check-a-long-for-null-in-java